com.arc.embeddedcdt/src/com/arc/embeddedcdt/gui/DebuggerGroupContainer.java

```diff
@@ -719,7 +719,7 @@ public void modifyText(ModifyEvent event) {
   }
 
   public static String getAttributeValueFromString(String string) {
-    if (string.length() > 0) {
+    if (string != null && string.length() > 0) {
```

Comment:
I wish you'd write slightly more clear messages, for example "Fix possible NPE". You can't fix a "null-check" because there was no "null-check" in the first place - you can fix only something that was there, but done wrongly. Here you actually add a null-check.